src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Circle | radius: " + String.format("%.2f", radius)
+                + " | area: " + String.format("%.2f", calculateArea())
+                + " | color: " + getColor());
```

Comment:
Just a small remark here: when using `String.format()` method, you can use a pattern with placeholders and pass real values as args - that way you write it in a single line or even a string block (since Java 12), which makes it easier to read and modify:

single line:
```
String.format("Figure: Circle, radius: %.2f, area: %.2f",  color: %s", radius, calculateArea(), getColor());
```